### PR TITLE
Add meeting 33 blog post

### PR DESCRIPTION
**Pull request by adelpopelkova**
This is the long article, there are a few things that need to be done:

- [x] Remove weird notes with question marks.
- [x] Add [links](https://encukou.github.io/freezeyt/). -> So that CI works. 🚀
- [x] Change quotation marks to : „“
- [x] Check that all code is marked as `code`.
- [x] Curious question - how does freezeyt behave with emoticons? I'll try it in my fork, but I need all links functioning, otherwise freeze will end up with an error.
- [x] What to do with emoticons in the article?
- [x] Check that the sentences that make sense.
- [x] Check the the facts are correct.
- [x] Add some reference to meeting 01 blog post - so that someone who would like to join would have these two blog posts together. They are the longest ones but at the same time they are the "most important" - the first one describes why did the project start and the basics, and this one should describe the current status of the project.

### CODE REVIEW

File: freezeyt_blog/articles/meeting33.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
 # Freezeyt sraz třicátý třetí - info pro nově příchozí
 
-## O čem projekt je?
+Chcete-li se k nám také připojit, doporučujeme přečíst si před tímto článkem,
+[článek z prvního srazu](../meeting01), kde je vysvětleno, co je cílem projektu.
```
Comment (adelpopelkova) [inline freezeyt_blog/articles/meeting33.md:4]: This part could be more motivational 😉

File: freezeyt_blog/articles/meeting33.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+
+Následně jsme se podívali na konfiguraci freezeytu.
+Tu můžeme dodat buď pomocí slovníku, nebo pomocí souboru.
+Jedna z důležitých věcí, kterou můžeme nakonfigurovat je `prefix` aplikace,
+ten zajistí, že odkazy budou odkazovat tam, kam mají.
+Také se dle něj dá poznat, zda je odkaz externí, nebo ne.
+Další možností konfigurace je to, kam se bude výsledek ukládat a to zda do
+adresáře na disk, nebo do slovníku. Ukládání do slovníku je hlavně kvůli testům,
+při ukládání do slovníku je tam také metoda, která kontroluje, že výsledek
+dostaneme zpět.
```
Comment (encukou) [inline freezeyt_blog/articles/meeting33.md:56]:
```suggestion
při ukládání do slovníku je tam také metoda, která umožní výsledek
dostat zpět.
```

File: freezeyt_blog/articles/meeting33.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+### Continuous Integration (CI)
+
+Když na GitHub pošleme PR, tak na něm GH pustí testy.
+Testujeme na 4 verzích Pythonu (3.6, 3.7, 3.8 a 3.9),
+pak tam jsou ještě testy s Pyflakes,
+které kontrolují většinou zapomenuté importy.
+Dále se automaticky zamrazí a publikuje blog.  
+
+Nastavení GH actions je v `.github/workflows`
```
Comment (encukou) [inline freezeyt_blog/articles/meeting33.md:119]:
```suggestion
Nastavení GH actions je v `.github/workflows`.
```

File: freezeyt_blog/articles/meeting33.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+### Další soubory
+
+Dále je v repozitáři nějaká dokumentace s poznámkami.
+Pak tam jsou nějaké Gitové soubory, licence, README a informace o závislostech.
+Ty se dělí na závislosti pro běh, pro vývoj a pro blog.
+
+Soubor `setup.py` z toho udělá Pythonní balíček a je tam ještě konfigrace Toxu,
+který slouží pro to, abychom testy mohly spustit na více verzích Pythonu.
+
+## TODO nadpis
```
Comment (encukou) [inline freezeyt_blog/articles/meeting33.md:130]:
```suggestion
## Blog
```

File: freezeyt_blog/articles/meeting33.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+Na chvíli jsme se pozastavili u toho, jak se píší články na blog.
+(Články na blog klidně přidávejte, Pull Requesty jsou velice vítány. 😉)
+Pokud nějaký článek píšete/chcete napsat,
+hlašte se [zde](https://github.com/encukou/freezeyt/issues/1) nebo na Slacku.
+
+Informace o tom, co všechno se dá nastavit v konfiiguraci by měla být napsána
```
Comment (encukou) [inline freezeyt_blog/articles/meeting33.md:136]:
```suggestion
## Popis konfigurace

Informace o tom, co všechno se dá nastavit v konfiguraci by měla být napsána
```

File: freezeyt_blog/articles/meeting33.md [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+$ export FLASK_ENV=developement
+$ flask run
+```
+
+Jen tak mimochodem jsme došli k tomu, že na blog by bylo fajn přidat styly!
+
+Pak jsme v README hledali (a našli), jak se freezeyt používá.
+Je potřeba přidat freezeyt do proměnné prostředí PYTHONPATH.
+Poté nainstalujeme závislosti (dependencies) freezeytu.
+Na statických stránkách je problém s přesměrováním. (Proč je tu tato věta?)
```
Comment (encukou) [inline freezeyt_blog/articles/meeting33.md:160]:
```suggestion
(Aplikace se kterou jsme to zkoušeli používala přesměrování, což server
statických stránek neumí.
Tak jsme udělali odbočku a přesměrování v této aplikaci zrušili.)
```

File: freezeyt_blog/articles/meeting33.md [hunk 7]
```diff
@@ -0,0 +1,246 @@
...
+Následně spustíme freezeyt.
+Potom „zmražené“ stránky nasadíme na server,
+nebo použijeme v Pythonu vestavěný `http.server`.
+V konfiguraci můžeme nastavit `freezeyt.freezing` na `True`, a to znamená,
+že některé části by se nezafreezovaly.
```
Comment (encukou) [inline freezeyt_blog/articles/meeting33.md:165]:
```suggestion
V konfiguraci můžeme nastavit `freezeyt.freezing` na `True`.
Na základě toho může aplikace např. vynechat dynamické stránky (a odkazy na ně).
```

File: freezeyt_blog/articles/meeting33.md [hunk 8]
```diff
@@ -0,0 +1,246 @@
...
+Následně je potřeba přepnout se do virtuálního prostředí.
+Zjistit, jak se daná aplikace importuje.
+Potom jsme nastavili PYTHONPATH (až bude balíček, bude to jednodušší).
+Pak by to mělo jít uložit. (Nemám sebemenší tušení, co by mělo jít uložit.)
+(Proč mám pocit, že tu je ta část dvakrát?)
```
Comment (encukou) [inline freezeyt_blog/articles/meeting33.md:170]:
```suggestion
Pak jde freezeyt spustit a aplikaci "zamrazit" – uložit na disk.
```

File: freezeyt_blog/articles/meeting33.md [hunk 9]
```diff
@@ -0,0 +1,246 @@
...
+### Článek z meetingu 32
+První PR přidával článek z minulého meetingu.
+Ten jsme začlenili v průběhu ukázky GH actions.
+
+### Rozepsané články na blog
+Dále byly na blog přidány historické draft PRs z předchozích srazů.
+Rozepsané články jsou lepší než nic.
+Po začlenění PR se články za chvíli objeví na [blogu](../).
+
+### Odebrní WSGI demo
```
Comment (encukou) [inline freezeyt_blog/articles/meeting33.md:188]:
```suggestion
### Odebrání WSGI demo
```

File: freezeyt_blog/articles/meeting33.md (hunk 8 above)
Comment (adelpopelkova) [reply freezeyt_blog/articles/meeting33.md:170]:
```suggestion
Pak jde freezeyt spustit a aplikaci „zamrazit" – uložit na disk.
```

File: freezeyt_blog/articles/meeting33.md [hunk 10]
```diff
@@ -0,0 +1,249 @@
...
+
+Chcete-li se k nám také připojit, doporučujeme přečíst si před tímto článkem,
+[článek z prvního srazu](../meeting01/), kde je vysvětleno, co je cílem projektu.
+
+🎉 Dneska se k nám připojili noví členové,
+tak jsme na úvod zařadili info o projektu a krátké demo jeho použití.
+
+## O čem projekt je?
+Abychom shrnuli o čem projekt je: první potřebujeme vytvořit webovou aplikaci
+(třeba ve webovém fremeworku [Flask](https://flask.palletsprojects.com/en/1.1.x/).
```
Comment (adelpopelkova) [inline freezeyt_blog/articles/meeting33.md:11]:
```suggestion
(třeba ve webovém fremeworku [Flask](https://flask.palletsprojects.com/en/1.1.x/)).
```

File: freezeyt_blog/articles/meeting33.md [hunk 11]
```diff
@@ -0,0 +1,249 @@
...
+Nastavení GH actions je v `.github/workflows`.
+
+### Další soubory
+
+Dále je v repozitáři nějaká dokumentace s poznámkami.
+Pak tam jsou nějaké Gitové soubory, licence, README a informace o závislostech.
+Ty se dělí na závislosti pro běh, pro vývoj a pro blog.
+
+Soubor `setup.py` z toho udělá Pythonní balíček a je tam ještě konfigrace Toxu,
+který slouží pro to, abychom testy mohly spustit na více verzích Pythonu.
```
Comment (adelpopelkova) [inline freezeyt_blog/articles/meeting33.md:128]:
```suggestion
který slouží pro to, abychom testy mohli spustit na více verzích Pythonu.
```

File: freezeyt_blog/articles/meeting33.md [hunk 12]
```diff
@@ -0,0 +1,249 @@
...
+Prohlížeč chybu nevyhodí, asi to bere jako relativní URL na tu konkrétní stránku.
+Ta prázdná adresa není přesně dle standardu, dělat by se to nemělo.
+My buď můžeme vyhodit chybu, nebo se chovat jako prohlížeč.
+Přidání testu zdůvodňuje rozhodnutí a zároveň to zaručí,
+že neměníme chování nástroje.
+
+### Specifikace stránek pomocí generátoru
+Poslední PR přidává možnost specifikovat extra stránky pomocí generátoru,
+i když na ně nevede žádný odkaz.
+Je to funkce která přidá extra pages, když je tam generátor tak jej spustí a přidá dané URL. (snad?)
```
Comment (adelpopelkova) [inline freezeyt_blog/articles/meeting33.md:220]:
```suggestion
Je to funkce která přidá extra pages, když je tam generátor tak jej spustí a přidá dané URL.
```

**Comment by adelpopelkova** [comment]
Thanks a lot for the review. Are the missing to-do items from the first comment OK?

**Comment by encukou** [comment]
All except "Add some reference to meeting 01 blog post " should be OK, once the suggestions are applied.

I wonder if the reference should be from the main page rather than from the first blog post?

**Comment by adelpopelkova** [comment]
>I wonder if the reference should be from the main page rather than from the first blog post?

I meant it in the way to link to the first blog post from this article.

**Comment by adelpopelkova** [comment]
> once the suggestions are applied.

Your suggestions are applied, It's a few commits up.

**Comment by encukou** [comment]
Ah! Then that's OK – the reference is there :)

**Review by encukou — APPROVED** [review]

**Pull request merged**